### PR TITLE
Add low bandwidth gpu training with dgc

### PR DESCRIPTION
**Pull request by wangxicoding**
preview
截图 2020-01-06 22 41 12-fullpage

### CODE REVIEW

**Review by gongweibao — APPROVED** [review]
LGTM

File: doc/fluid/advanced_usage/best_practice/gpu_training_with_low_bandwidth_dgc.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# 低配网络的分布式GPU训练
+
+## 1. 背景
+  大规模分布式训练需要较高的网络带宽以便进行梯度的聚合更新，这限制了多节点训练时的可扩展性同时也需要昂贵的高带宽设备。在低带宽云网络等环境下进行分布式训练会变得更加糟糕。现有[Deep Gradient Compression](https://arxiv.org/abs/1712.01887)研究表明，分布式SGD中有99.9的梯度交换都是冗余的，可以使用深度梯度压缩选择重要梯度进行通信来减少通信量，降低对通信带宽的依赖。Paddle目前实现了DGC的稀疏通信方式，可有效在低配网络下进行GPU分布式训练。下面将介绍DGC稀疏通信方式的使用方法、适用场景及基本原理。
```
Comment (Boyan-Liu) [inline doc/fluid/advanced_usage/best_practice/gpu_training_with_low_bandwidth_dgc.md]: 99.9是指99.9%？

Comment (wangxicoding) [reply doc/fluid/advanced_usage/best_practice/gpu_training_with_low_bandwidth_dgc.md]: Done. 99.9%

**Review by Boyan-Liu — APPROVED** [review]
LGTM

**Pull request merged**